xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.h

```diff
@@ -73,7 +72,8 @@ class CRenderPicture
 public:
   CRenderPicture(CSurfaceContext *context);
   ~CRenderPicture();
-  ID3D11View*      view;
+  ID3D11View* view;
+  unsigned int extFormat;
```

Comment:
Can you please change `extFormat` type to DXGI_FORMAT and rename it to `format`? then it will be correspond to my current work on this part.